Project management/SEO preparation.md

```diff
@@ -0,0 +1,163 @@
+
+```
+
+
+This will add the sitemap to the robots file. A sitemap is generated by the Yoast plugin and is a crucial way to tell search engine crawlers which pages should be crawled.
+
+Some pages, like search result pages and Thank you pages (form submissions Thank you pages), shouldn't be indexed. Yoast and WordPress should add the necessary `<meta name="robots" content="noindex, follow">` tag that won't index search result pages. But make sure it's there when testing the site.
+
+When creating custom post types, check if they will be of a specific type. By that, we mean things like FAQ posts, recipes, events, products, etc. because they should have specific structured data associated with them.
+
+#### Basic checklist
+
+**Before deploying** to production make sure to check that:
+
+- Posts and pages have correct OG images set
+- Remove noindex option (blocking search engines from the WordPress settings)
+- Check if canonical links are set correctly
+- Check if the domains are consolidated (work with DevOps engineers on this one)
+  - http to https redirection is set
+  - correct www version is set (site should either have www or not, and redirections should be set)
+  - Trailing slashes - choose either with or without it
+- Check that sitemap doesn't have 301 links in it
+- Optimize images, especially on the main pages
+
+In the case, the site was on another domain and a new one is being set up it's crucial to check the redirections and if they are correct. For redirections, we are using the [Redirection plugin](https://wordpress.org/plugins/redirection/).
+
+Once we check for the errors using automated tools (Lighthouse, Screaming Frog, Ahrefs, Semrush) we should fix those and resubmit the new `sitemap.xml` in Google Search Console (so it can be reindexed).
+
+### The existing website (content migration)
+
+We won't be covering the content migration itself here, as that is covered in the other chapter of the handbook (link will be added).
+
+Here we'll cover things that developers need to be careful about when doing the migration.
+
+Before doing the migration we should **work with the Growth team** in detail so that they can get all the relevant information about the existing site.
+
+They will give us information about the site's health (number of dead links, existing SEO ranking, top 50 pages/posts on the old website, etc.)
+
+When migrating the content it's crucial to **get all the old links** from the website associated with the new website. That means that the old permalink should be added to the migration data, as well as some extra identifiers such as old post ID, and stored as metadata for every imported post.
+
+This will allow us to easily fetch all the new and old permalinks that we can easily add to the redirect list. Also, it would be good that the **old permalinks that had the biggest SEO score** on the old website **remain the same** on the new one if possible (if not prioritize that their redirects are set up correctly).

```

Comment:
```suggestion
This will allow us to easily fetch all the new and old permalinks that we can easily add to the redirect list. Also, it would be good that the **old permalinks that had the biggest SEO score** on the old website **remain the same** on the new one if possible (if not, prioritize that their redirects are set up correctly).

```